snap/snapcraft.yaml

```diff
@@ -186,6 +186,7 @@ parts:
       - net-tools
       - openssl
       - procps
+      - python3-jsonschema
```

Comment:
isn't jsonschema already in the dependencies? https://github.com/ubuntu/microk8s/blob/93acc1bb4176b4ce3128a74990ad5fb5abebd0d8/snap/snapcraft.yaml#L266